git-cliff-core/src/repo.rs

```diff
@@ -310,6 +310,17 @@ impl Repository {
 		None
 	}
 
+	fn should_include_tag(
```

Comment:
Can you explain this function a bit?

Are you simply checking if a commit belongs to a branch?

A descriptive comment would be really helpful.

Reply:
sure updated